utils/yoke/src/yoke.rs

```diff
@@ -74,14 +74,37 @@ use alloc::sync::Arc;
 /// assert_eq!(&**yoke.get(), "hello");
 /// assert!(matches!(yoke.get(), &Cow::Borrowed(_)));
 /// ```
-#[derive(Debug)]
 pub struct Yoke<Y: for<'a> Yokeable<'a>, C> {
     // must be the first field for drop order
     // this will have a 'static lifetime parameter, that parameter is a lie
     yokeable: Y,
     cart: C,
 }
 
+impl<Y: for<'a> Yokeable<'a>, C: core::fmt::Debug> core::fmt::Debug for Yoke<Y, C>
```

Comment:
not: please mention that this cannot be an autoderive due to soundness issues with the derive being able to call a `'static` impl

Reply:
I’ve added a comment